tests/FunctionalClientTest.php

```diff
@@ -216,7 +214,7 @@ public function testExecStreamOutputInMultipleChunksWhileRunning($container)
         $stream->once('data', $this->expectCallableOnceWith('hello'));
         $stream->on('end', $this->expectCallableOnce());
 
-        $output = Block\await(Stream\buffer($stream), Loop::get());
+        $output = \React\Async\await(Stream, Loop::get());
```

Comment:
Was getting rid of `\buffer($stream)` intentional? There's also a `Loop::get()` leftover.